src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -383,31 +390,47 @@ private void UpdatePatchDetails()
             percentageFormat.FormatSafe(Math.Round(GetCompoundAmount(SelectedPatch, carbondioxideCompound),
                 Constants.ATMOSPHERIC_COMPOUND_DISPLAY_DECIMALS));
 
+        var otherAmount = CalculateOtherCompoundAmount();
+
+        otherCompoundLabel.Text = percentageFormat.FormatSafe(otherAmount,
+            Constants.ATMOSPHERIC_COMPOUND_DISPLAY_DECIMALS);
+
+        if (SelectedPatch != null)
+        {
+            if (otherAmount <= 0)
+            {
+                otherCompoundBase.Visible = false;
+            }
+            else
+            {
+                otherCompoundBase.Visible = true;
+            }
+        }
+        else
+        {
+            otherCompoundBase.Visible = false;
+        }
+
         // Compounds
         hydrogenSulfideLabel.Text =
-            Math.Round(GetCompoundAmount(SelectedPatch, hydrogensulfideCompound),
-                    Constants.PATCH_CONDITIONS_COMPOUND_DISPLAY_DECIMALS)
-                .ToString(CultureInfo.CurrentCulture);
+            Math.Round(GetCompoundAmount(SelectedPatch!, hydrogensulfideCompound),
```

Comment:
These seem like unnecessary changes that you should revert. If you remove the above null check the automatic checks should stop complaining about null problems here.